CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 
 ### Enhancements
 * Support setting `maxNumberOfActiveVersions` when creating a `Configuration`. ([#1036](https://github.com/realm/realm-dart/pull/1036))
+* Add ManagedRealmList.move method that allows notifications to correctly report moves, as opposed to reporting moves as deletes + inserts. ([#1037](https://github.com/realm/realm-dotnet/issues/1037))
```

Comment:
realm-dotnet?

Reply:
Also while this is technically true move also rearranges items. Why not mention the real feature here. Isnt the notifications more like a side effect of this feature

Reply:
I agree, I should also mention that the element is actually moved 😄 

How about:
> * Add List.move extension method that moves an element from one index to another. Delegates to ManagedRealmList.move for managed lists. This allows notifications to correctly report moves, as opposed to reporting moves as deletes + inserts. ([#1037](https://github.com/realm/realm-dart/issues/1037))

I do kind of think the notification part is the important feature, since that is the reason we don't just implement `move` as:
```dart
void move(int from, int to) => insert(to, removeAt(from));
```

Reply:
it is good to mention the notification as well